### PR TITLE
account usage api support added

### PR DESCRIPTION
**Pull request by stopal-r7**
account usage api added.
minor documentation problems fixed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -160,43 +160,43 @@ The user and account management functionality of the CLI can only be used with a
 It is worth noting that if your account does not have a specific owner set then some user management functions may fail with a 500 error; to check if an owner is set the 'getowner' command below can be used. If no owner is set then you must regenerate the owner API key, at which point you will be asked to set an account owner.
 
 ####List Users
-The 'userlist' command will return a list of all users that have access to the account for which the CLI has been configured. The command will return the users first and last name, email address, user key and the last time they logged in. The 'userlist' command does not accept any arguments.
+The 'listusers' command will return a list of all users that have access to the account for which the CLI has been configured. The command will return the users first and last name, email address, user key and the last time they logged in. The 'userlist' command does not accept any arguments.
```
Comment (pclune-r7) [inline README.md:164]: 'userlist' still appears in this sentence, should be new command

File: README.md [hunk 2]
```diff
@@ -160,43 +160,43 @@ The user and account management functionality of the CLI can only be used with a
...
 
 An existing user can be added to your account use the following command
 ```
-lecli useradd -u 12345678-aaaa-bbbb-1234-1234cb123456
+lecli adduser -u 12345678-aaaa-bbbb-1234-1234cb123456
 ```
 
 ####Delete User
-To userdel command allow for the removal of a user from your account and deletion of the users account from Logentries. 
+To 'deleteuser' command allow for the removal of a user from your account and deletion of the users account from Logentries.
```
Comment (pclune-r7) [inline README.md]: change

The 'deleteuser' command allows for the removal of a user from your account and deletion of the users account from Logentries.

**Comment by pclune-r7** [comment]
the code and tests LGTM - just a couple of minor fixes in the READ.md file - once done then......

+1

**Review by pclune-r7 — APPROVED** [review]

**Review by danilosterrapid7 — APPROVED** [review]
LGTM

**Pull request merged**